Mathlib/Algebra/Group/UniqueProds/Basic.lean

```diff
@@ -67,6 +67,10 @@ namespace UniqueMul
 
 variable {G H : Type*} [Mul G] [Mul H] {A B : Finset G} {a0 b0 : G}
 
+@[to_additive]
+theorem mono {A' B' : Finset G} (hA : A ⊆ A') (hB : B ⊆ B') (h : UniqueMul A' B' a0 b0) :
+    UniqueMul A B a0 b0 := fun _ _ ha hb he ↦ h (hA ha) (hB hb) he
+
```

Comment:
This one doesn't really belong in this PR, but it's trivial enough that it doesn't matter too much; if you keep it, please mention it in the PR description (explaining that it works towards the other PR)